src/sql/azdata.proposed.d.ts

```diff
@@ -2068,9 +2068,14 @@ declare module 'azdata' {
 		 * @param connectionUri The URI of the connection to the target server
 		 * @param targetPath The file path on the server machine to open by default in the dialog
 		 * @param fileFilters The filters used to limit which files are displayed in the file browser
+		 * @param showFoldersOnly Optional argument to specify whether the browser should only show folders
 		 * @returns The path of the file chosen from the dialog, and undefined if the dialog is closed without selecting anything.
 		 */
-		export function openServerFileBrowserDialog(connectionUri: string, targetPath: string, fileFilters: FileFilters[]): Thenable<string | undefined>;
+		export function openServerFileBrowserDialog(connectionUri: string, targetPath: string, fileFilters: FileFilters[], showFoldersOnly?: boolean): Thenable<string | undefined>;
+	}
+
+	export interface FileBrowserProvider extends DataProvider {
+		openFileBrowser(ownerUri: string, expandPath: string, fileFilters: string[], changeFilter: boolean, showFoldersOnly?: boolean): Thenable<boolean>;
```

Comment:
Did this never get the doc comment added to it? Please follow up with doing that in main....(it doesn't need to be ported over here)

Reply:
Woops, merged the PR after I got home and forgot about that. I'll update it in main

Reply:
@Charles-Gagnon https://github.com/microsoft/azuredatastudio/pull/24413